perfkitbenchmarker/benchmark_spec.py

```diff
@@ -387,8 +393,10 @@ def DeleteVm(self, vm):
 
   def PickleSpec(self):
     """Pickles the spec so that it can be unpickled on a subsequent run."""
+    flags, self.FLAGS = self.FLAGS, None
```

Comment:
OK. Can you note that here?
Are the FLAGS correctly serialized this way? Or do the same flags need to be specified when the spec is deserialized?